src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+interface TabsProps {
+  onTabSelected: (idTab: string) => void;
+  selectedTabId: {
+    title: string;
+    content: string;
+  };
+  tabs: {
+    id: string;
+    title: string;
+    content: string;
```

Comment:
hmm, why do you need a selectedTabId if it's almost a copy of a tab?
You can just make like this

1) create a separated file types and move common Tab interface there
```
interface Tab {
  id: string;
  title: string;
  content: string;
}
```

2) import Tab interface here an use for props
```
type Props = {
  tabs: Tab[],
  setSelectedTab: React.Dispatch<React.SetStateAction<Tab>>,
  selectedTabId: string,
};

```